apps/macOS/FluentUIReactNativeTester_macOS/Config/FluentUIReactNative_common.xcconfig

```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+PRODUCT_MODULE_NAME = FluentUIReactNativeTester
+SWIFT_MODULE_NAME = FluentUIReactNativeTester
+
+ALWAYS_SEARCH_USER_PATHS = NO
+CLANG_ENABLE_MODULES = YES
+
+// Versions
+MACOSX_DEPLOYMENT_TARGET = 10.14
```

Comment:
Our devmain apps need to still support 10.13. Suspect we'll want to match that 'N-2' support on macOS, 'N-1' support on iOS (N = current OS version)